docs/source/nlp/question_answering.rst

```diff
@@ -163,139 +136,76 @@ evaluation:
-+-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
-| **test_ds.file**                          | string          | The test file path (optional).                                                                               |
-+-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
-| **test_ds.num_samples**                   | integer         | The number of samples to use from the test dataset (use ``-1`` to specify all samples).                      |
-+-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
-
-Example of the command for training the model:
-
-.. code::
-
-    python question_answering_squad.py \
-           model.train_ds.file=<PATH_TO_TRAIN_DATA_FILE>  \
-           model.validation_ds.file=<PATH_TO_VALIDATION_DATA_FILE>  \
-           model.dataset.version_2_with_negative=<ALLOW_UNANSWERABLE_SAMPLES>  \
-           model.dataset.do_lower_case=<DO_LOWER_CASE> \
-           trainer.max_epochs=<NUM_EPOCHS> \
-           trainer.devices=[<CHANGE_TO_GPU(s)_YOU_WANT_TO_USE>] \
-           trainer.accelerator='gpu'
-
-.. Note:: 
-  
-   The first time you train, it will take an extra 5-10 minutes to process the dataset. For future training runs, it will use the 
-   processed dataset if :code:`model.dataset.use_cache=true`, which is automatically cached in the files in the same directory as
-   the data.
-
-Required Arguments for Training
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-- :code:`model.train_ds.file`: path to the training file in JSON format
-- :code:`model.validation_ds.file`: path to the validation file in JSON format
-
-Fine-tuning Procedure
-^^^^^^^^^^^^^^^^^^^^^
-
-Fine-tuning procedure and logs look similar to what's described in the Model Training section, with the addition of the model
-that is initially loaded from a previously trained checkpoint, e.g. by specifying :code:`pretrained_model=<PRETRAINED_MODEL_NAME>`.
-
-Inference
----------
-
-An example script on how to run inference can be found at `examples/nlp/question_answering/question_answering_squad.py <https://github.com/NVIDIA/NeMo/blob/stable/examples/nlp/question_answering/question_answering_squad.py>`_.
-
-To run inference with the pre-trained model, run:
+For downloading the MS-MARCO dataset, Terms of Use need to be accepted at https://microsoft.github.io/msmarco/. The dataset files can be downloaded from https://msmarco.blob.core.windows.net/msmarco/train_v2.1.json.gz and https://msmarco.blob.core.windows.net/msmarco/dev_v2.1.json.gz.
```

Comment:
Can you add something on if you have other datasets (give some examples based those in the MultiQA 7 datasets) , you should convert into squad format and in a nested json structure how that squad format looks like?

Reply:
This is actually already in the previous document - can you copy that over?